packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -790,7 +770,37 @@ export class ApolloServerBase {
     // at the end of that list so they take precidence.
     // A follow-up commit will actually introduce this.
 
+    // Enable cache control unless it was explicitly disabled.
+    if (this.config.cacheControl !== false) {
```

Comment:
With `cacheControl` implemented as a plugin now, do you anticipate we'll remove this configuration point in a future major (v4)?

Reply:
Possibly.  I feel like our `cacheControl` integration might get tighter over time, but it's certainly possible we could have configuration be entirely composable via plugins.  I don't imagine we'd sacrifice this particular convenience plugin initialization anytime soon though.